runelite-client/src/main/java/net/runelite/client/plugins/groundmarkers/GroundMarkerConfig.java

```diff
@@ -45,4 +44,14 @@ default Color markerColor()
 	{
 		return Color.YELLOW;
 	}
+
+	@ConfigItem(
+		keyName = "usePerTileColors",
+		name = "Use per-tile colors",
+		description = "Color tiles using the color from time of placement"
+	)
+	default boolean rememberTileColors()
```

Comment:
This should be defaulted to false, imo.

Reply:
I disagree, but I'm not going to revert your change.